releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,14 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.0.0 -
+
+### Enhancements
+* Added an enhancement that supports using custom service accounts for workload identity authentication. This is useful for permission management isolation between different namespaces.

```

Comment:
Does this make sense?

Added support for binding a managed identity with a custom service account within a namespace. This feature enhances security isolation by namespaces when using workload identity authentication.